docs/source/index.rst

```diff
@@ -23,8 +23,9 @@ The explicit code interfaces and structure are described in the :ref:`code-docum
    development-guide/index
    project-management/index
    external-tools/index
+   rest-api/index
```

Comment:
```suggestion
   api/index
```
I'd suggest leaving "rest" out of it, but that's not a super strong preference

Reply:
Data API? Or some other qualifier if we want to be more explicit what it is an API for. I agree, REST is how it functions, but not descriptive.

Reply:
I like "Data API" or perhaps "Data Access API"